### ISSUE TITLE
Error in loading hubs when self hosted

### ISSUE THREAD

**Issue by shashankgwl** [SignalR/SignalR#941]
Hello,

SERVER : 
This is my self hosted server , running inside a console application.

Console.WriteLine("Chat server starting...");
var server = new Server("http://10.86.89.101:8081/");
server.MapHubs();
server.Start();
Console.WriteLine("Press any key to stop ....");
Console.ReadLine();

CLIENT :

I'm successfully able to connect my javascript client on my machine using this simple code

<script src="http://10.86.89.101:8081/signalr/hubs"></script>

var chatClient = $.connection.myHub;
            $.connection.hub.start(
                function ()
                {
                    chatClient.addMember(myName);
                }
            );

In short everything is working fine on my machine, 
PROBLEM : When my friend tries to run the same client code from his machine he gets the error
"SignalR: Error loading hubs. Ensure your hubs reference is correct, e.g. <script src='/signalr/hubs'></script>."

ALSO, he's not able to access the proxy URL which I'm able to access i.e. http://10.86.89.101:8081/signalr/hubs

MY MACHINE is accessible from his machine, if I host an application on my iis he can access it, it's only the proxy hub url which he can't access, 

any help will be gr8 for me.
